### PR TITLE
rename AwsChaos (GcpChaos) to AWSChaos (GCPChaos)

### LINKED ISSUE #2014: We need use upper case with some name of chaos
## Question

As https://github.com/golang/go/wiki/CodeReviewComments#initialisms says, we should keep the consistent case with words: AWS, GCP, etc. And we have rename `IoChaos` to `IOChaos`

Related chaos:

- AwsChaos -> AWSChaos
- GcpChaos -> GCPChaos

### PR DESCRIPTION
**Pull request by fewdan**
Signed-off-by: “fewdan” <fewdan@hotmail.com>

### What problem does this PR solve?

Issue Number: close #2014  

Problem Summary:

### What is changed and how it works?

rename AwsChaos (GcpChaos) to AWSChaos (GCPChaos)

Proposal: [xxx](url) 

What's Changed:

### Related changes

* PR to update `chaos-mesh/website`/`chaos-mesh/website-zh`:
* Need to update Chaos Dashboard component, related issue:
* Need to cheery-pick to the release branch

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

### Release note 

```release-note
Please add a release note.
If you don't think this PR needs a release note then fill it with None.
```

### CODE REVIEW

File: controllers/chaosimpl/awschaos/ec2stop/impl.go [hunk 1]
```diff
@@ -36,13 +36,13 @@ type Impl struct {
 }
 
 func (impl *Impl) Apply(ctx context.Context, index int, records []*v1alpha1.Record, obj v1alpha1.InnerObject) (v1alpha1.Phase, error) {
-	awschaos := obj.(*v1alpha1.AwsChaos)
+	awschaos := obj.(*v1alpha1.AWSChaos)
 
-	var selected v1alpha1.AwsSelector
+	var selected v1alpha1.AWSSelector
 	json.Unmarshal([]byte(records[index].Id), &selected)
```
[the remark this answers is not in the document]
Comment (fewdan) [reply controllers/chaosimpl/awschaos/ec2stop/impl.go:42]: ignore

**Comment by fewdan** [comment]
@g1eny0ung 
I'm not very familiar with "ts" and "tsx", so please take a look at what else needs to be modified when it is convenient for you?

**Review by g1eny0ung — APPROVED** [review]
LGTM

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**